src/convertToObject.js

```diff
@@ -5,8 +5,22 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styleArr = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style !== '');
+
+  styleArr.forEach((item) => {
+    const [property, value] = item.split(':').map((style) => style.trim());
+
+    styleObject[property] = value;
+  });
+
+  return styleObject;
```

Comment:
The function `convertToObject` is implemented correctly, but there's an opportunity to improve the code style. The chained method calls starting from line 12 should have the same indentation to enhance readability.